microservices/payment-stripe/src/services/card.ts

```diff
@@ -176,7 +176,7 @@ class Card {
     manager: EntityManager,
     { data: { isFromWebhook } }: QueryRunner,
   ): Promise<void> {
-    const service = await Factory.create(getManager());
+    const service = await Stripe.init(getManager());
```

Comment:
maye move to getStripe method with optional parameter manager

Reply:
@MatthewPattell Like this? 
<img width="811" alt="image" src="https://github.com/Lomray-Software/microservices/assets/44732463/be15f13a-d651-444a-aa9e-99fd67f5aecc">

Reply:
no, forgot it